lib/pluginmanager/main.rb

```diff
@@ -32,6 +33,7 @@ class Main < Clamp::Command
       subcommand "pack", "Package currently installed plugins", LogStash::PluginManager::Pack
       subcommand "unpack", "Unpack packaged plugins", LogStash::PluginManager::Unpack
       subcommand "list", "List all installed plugins", LogStash::PluginManager::List
+      subcommand "new", "Create a new plugin structure", LogStash::PluginManager::New
```

Comment:
@purbon can we call this `generate` and make the test `Create the foundation for a new plugin`